deployment/README.md

```diff
@@ -19,7 +19,7 @@
 A collection of scripts for making life easier.
 
 ## [hr-data-generator](../hr-data-generator/README.md)
```

Comment:
Does this header need to be changed? There isn't a README for the hr-data-generator

Reply:
Agreed, needs to be changed to a full github URL

Reply:
updated